dev-requirements.txt

```diff
@@ -1,54 +1,6 @@
-alabaster==0.7.6
-apache-libcloud==0.20.1
-astroid==1.4.4
-awscli==1.9.17
-backports.ssl-match-hostname==3.4.0.2
-botocore==1.3.17
-cached-property==1.2.0
-chardet==2.3.0
-colorama==0.3.6
-coloredlogs==1.0.1
-deb-pkg-tools==1.34
-docutils==0.12
-ecdsa==0.13
-executor==4.4
-Fabric==1.10.2
-flake8==2.4.1
-gitdb==0.6.4
-GitPython==1.0.1
-humanfriendly==1.33
-Jinja2==2.7.3
-jmespath==0.9.0
-lazy-object-proxy==1.2.1
-MarkupSafe==0.23
-mccabe==0.3.1
-ndg-httpsclient==0.4.0
-packaging==15.2
-paramiko==1.15.2
-pep8==1.5.7
-pg8000==1.10.2
-pluggy==0.3.0
-py==1.4.29
-pyenchant==1.6.6
-pyflakes==0.8.1
-Pygments==2.0.2
-pylint==1.5.4
-pymongo==3.0.2
-PyMySQL==0.6.6
-python-dateutil==2.4.2
-python-debian==0.1.23
-requests-file==1.3.1
-smmap==0.9.0
-snowballstemmer==1.2.0
-Sphinx==1.3.1
-sphinx-prompt==1.0.0
-sphinx-rtd-theme==0.1.8
-sphinxcontrib-httpdomain==1.3.0
-sphinxcontrib-spelling==2.1.2
-testtools==2.0.0
-tox==2.1.1
-troposphere==1.3.0
-six==1.10.0
-versioneer==0.15
-virtualenv==13.1.0
-wrapt==1.10.6
```

Comment:
I've turned this file into a shortcut for installing everything + Flocker in editable mode.